packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail/RancherDetail.component.tsx

```diff
@@ -89,11 +90,28 @@ const RancherDetail = ({
 
   const onAccessRancherUrl = () =>
     trackAction(TrackingPageView.DetailRancher, TrackingEvent.accessUi);
-
+  const { i18n } = useTranslation('common');
   const shouldDisplayUpdateSoftware =
     getLatestVersionAvailable(rancher, versions) &&
     isReadyStatus &&
     !updateSoftwareResponseType;
+  const locales = useRef({ ...dateFnsLocales }).current;
+  const userLocale = getDateFnsLocale(i18n.language);
+
+  const displayDate = useCallback(
+    (value: string) => {
+      if (userLocale in locales) {
+        const localeId = userLocale as keyof typeof locales;
+        return format(new Date(dateUsage), value, {
+          locale: locales[localeId],
```

Comment:
Isn't date fns supposed to provide a default format ?

Reply:
I'm not entirely sure to understand ? What kind of default format ?